bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/niri/Niri.java

```diff
@@ -550,12 +536,12 @@ public String getRecommendedMinExp() {
         public static final Mask DEFAULT = MASK_IMAGING;
 
         // The internal values of height and width for this instance of Mask
-        private double _width = Size.IMAGING_SIZE;
-        private double _height = Size.IMAGING_SIZE;
-        private String _displayValue;
-        private String _logValue;
+        private final double _width;
```

Comment:
Correct. There wasn't a use of the default values for these fields, the constructor of `Mask` forces you to initialize them. I actually realized that just by marking the items final and seeing what the compiler said.